src/jit/morph.cpp

```diff
@@ -1579,6 +1595,8 @@ void fgArgInfo::ArgsComplete()
                 // Spill multireg struct arguments that have Assignments or Calls embedded in them
                 curArgTabEntry->needTmp = true;
             }
+#ifndef _TARGET_ARM_
+            // TODO: This implementation is not tested on ARM32
             else
```

Comment:
Instead of `#ifdef`ing this out, can you make it a `NYI_ARM("multireg struct arguments");` ?